_episodes/12-cluster.md

```diff
@@ -46,6 +47,13 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the internet. We use a program called the Secure SHell or ssh for this. (make sure you have 
+this program in your laptop and refer the *setup* section for more details).
+
+{% include figure.html url="" max-width="50%" file="/fig/connect-to-remote.svg"
```

Comment:
Replace 'Internet' with 'Internet/network' (or just 'network') in this diagram?

Reply:
a544bb2

Reply:
f2414d1